devimint/src/external.rs

```diff
@@ -711,16 +692,21 @@ pub async fn open_channel_between_gateways(
         )
         .await?;
 
-    gw_cln
-        .open_channel(lnd_pubkey.clone(), 10_000_000, Some(5_000_000))
-        .await?;
+    poll("fund channel", || async {
+        gw_cln
+            .open_channel(lnd_pubkey.clone(), 10_000_000, Some(5_000_000))
+            .await
+            .map_err(ControlFlow::Continue)
+    })
+    .await?;
```

Comment:
What are we waiting for here? Why would it fail and later work? Maybe we want to wait for the node being up instead?

Reply:
We're polling here because that's what the existing `open_channel()` devimint function does: https://github.com/fedimint/fedimint/blob/e694b7e4d18b4815d4dc27d37db5b6789fcd75b6/devimint/src/external.rs#L582-L606

I added a TODO comment to remove the polling in `open_channel_between_gateways()`. Do you mind if I address that in a follow-up?